recipes/lidar/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "lidar" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0f6059a915c2f20668e74e347592171c3f897be90e69aea5ccf2c8e7707e1ec3
+
+build:
+  number: 0
+  entry_points:
+    - lidar=lidar.cli:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click >=6.0
+    - matplotlib
```

Comment:
If your package doesn't depend upon interactive images, but rather uses `matplotlib` to render images to files, then you should use `matplotlib-base` instead. `matplotlib` gives you the full Qt backend and consumes lots of space.

Reply:
Thanks for the suggestion! I have changed it to `matplotlib-base`